staging/src/k8s.io/noderesourcetopology-api/manifests/crd-instance-example.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: node1
+topologyPolicies:
```

Comment:
Where are these policies enumerated and described?

Reply:
These policies are populated by the on-node agent and are a reflection of the configured Topology Manager configuration on Kubelet running on that node.

Reply:
We need to make `topologyPolicies` more descriptive as we are currently coalescing the Topology manager policy and the scope. Recently Topology Manger policy Options were introduced as well so we need to capture those as well.